underscore.js

```diff
@@ -1203,7 +1205,10 @@
 
   // Perform a deep comparison to check if two objects are equal.
   _.isEqual = function(a, b) {
-    return eq(a, b);
+      if (a === b) return a !== 0 || 1 / a === 1 / b;
+      var typeA = typeof a;
+      return typeA != 'function' && typeA != 'object' && typeof b != 'object' ?
+        a != a && b != b : a != null && b != null &&  deepEq(a, b);
```

Comment:
Of course it's not. _.isEqual(null, null) will go through 1208, which in fact is the first line of the original eq function unchanged.

Sometimes it's so hard to find a valid reason to close PR, right? :)